routers/api/admin/Addmovies.js

```diff
@@ -1,17 +1,18 @@
 const models=require('../../../models');
+require('dotenv').config();
 const MovieDb = require('moviedb-promise')
 const moviedb = new MovieDb(process.env.TMDB_API_KEY)
+const Logger = require('../../../services/logger')
+const logger = new Logger('addmovie')
 async function addMovies(req,res,next)
 {
     try{
-         const movieimgobj = await moviedb.searchMovie({ query: req.body.name })
-         console.log(movieimgobj.results[0])
-        // Console.log(movieimgobj.results[0].poster_path)
+         const movieImgObj = await moviedb.searchMovie({ query: req.body.name })
           const movie=await models.Movies.create({
              name:req.body.name,
              releaseYear:req.body.releaseYear,
-             rating:parseInt(movieimgobj.results[0].vote_average,10),
-             imgURL:"https://image.tmdb.org/t/p/w185"+movieimgobj.results[0].poster_path
+             rating:parseInt(movieImgObj.results[0].vote_average,10),
```

Comment:
Never hard code values like this, use underscore/lodash module

Reply:
done